backend/package.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Pouruqi un package.json. C'est utile dans le futur ?

Reply:
It'll be useful when we want to use dependencies other than `aws-sdk` in the backend (which is automatically included in the context of all AWS lambdas).

But yeah technically it's not needed now. Removing it